index.bs

```diff
@@ -128,7 +140,25 @@ This callback will be invoked when there are changes to <i>target</i>'s intersec
     </pre>
 
     <div dfn-type="method" dfn-for="IntersectionObserver">
-        : <dfn>observe()</dfn>
+        : <dfn>Constructor(callback, options)</dfn>
```

Comment:
Note, tho, a constructor isn't a method. You need `<dfn constructor lt="IntersectionObserver(callback, options)">...</dfn>`.